README.md

```diff
@@ -28,7 +28,7 @@ Please refer to [notes](https://github.com/HewlettPackard/oneview-golang/blob/ma
 HPE OneView SDK for Go can be installed from Source or Docker container installation methods. You can either use a docker container which will have the HPE OneView SDK for Go installed or perform local installation.
 
 ###  Docker Setup
-The light weight containerized version of the HPE OneView SDK for Ruby is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/hpe-oneview-sdk-for-ruby). The Docker Store image tag consist of two sections: <sdk_version-OV_version>
+The light weight containerized version of the HPE OneView SDK for Go is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/hpe-oneview-sdk-for-golang). The Docker Store image tag consist of two sections: <sdk_version-OV_version>
```

Comment:
We can use this url https://hub.docker.com/r/hewlettpackardenterprise/hpe-oneview-sdk-for-golang